plugins/chat/spec/services/chat/update_message_spec.rb

```diff
@@ -510,8 +528,7 @@ def create_chat_message(user, message, channel, upload_ids: nil)
       old_message = "It's a thrsday!"
       new_message = "Today is Thursday, it's almost the weekend already!"
       chat_message = create_chat_message(user1, old_message, public_chat_channel)
-      updater =
-        described_class.call(guardian: guardian, message_id: chat_message.id, message: new_message)
+      described_class.call(guardian: guardian, message_id: chat_message.id, message: new_message)
```

Comment:
unrelated to the commit, but updated variable is not used (we need the call though)